Sources/TuistKit/Services/InitService.swift

```diff
@@ -1,266 +1,271 @@
 import FileSystem
+import Noora
 import Path
 import ServiceContextModule
-import TuistCore
-import TuistLoader
-import TuistScaffold
+import TuistServer
 import TuistSupport
-import XcodeGraph
 
-enum InitServiceError: FatalError, Equatable {
-    case ungettableProjectName(AbsolutePath)
-    case nonEmptyDirectory(AbsolutePath)
-    case templateNotFound(String)
-    case templateNotProvided
-    case attributeNotProvided(String)
-    case invalidValue(argument: String, error: String)
+public struct InitService {
```

Comment:
Let's rename to `InitCommandService` as suggested in the `inspect build` command PR?